schematic_api/api/routes.py

```diff
@@ -556,6 +559,7 @@ def get_viz_tangled_tree_text(schema_url, figure_type, text_format):
     
     return text_df
 
+@cross_origin(["http://localhost:8000/main.html", "https://sage-bionetworks.github.io/schema_visualization/"])
```

Comment:
These should be domain names, not full URL to resources.

```suggestion
@cross_origin(["http://localhost", "https://sage-bionetworks.github.io"])
```